packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -94,6 +95,14 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
        */
       path: String,
 
+      /**
+       * A comma-separated list of alternative paths matching this item.
+       */
+      pathAliases: {
+        type: String,
+        reflectToAttribute: true,
```

Comment:
When removing this line, all the tests still pass. We probably don't need it after all: users are not supposed to check for presence of the attribute anyways. Consider removing it altogether.

Reply:
Removed.